File: app/src/main/java/info/nightscout/androidaps/plugins/pump/insight/satl/VerifyConfirmRequest.java
package info.nightscout.androidaps.plugins.pump.insight.satl;

import info.nightscout.androidaps.plugins.pump.insight.ids.PairingStatusIDs;
import info.nightscout.androidaps.plugins.pump.insight.utils.ByteBuf;

public class VerifyConfirmRequest extends SatlMessage {

    @Override
    protected ByteBuf getData() {
        ByteBuf byteBuf = new ByteBuf(2);
        byteBuf.putUInt16LE(PairingStatusIDs.IDS.getID(PairingStatus.CONFIRMED));
        return byteBuf;
    }
}

File: insight/src/main/java/info/nightscout/androidaps/plugins/pump/insight/satl/VerifyConfirmRequest.kt
package info.nightscout.androidaps.plugins.pump.insight.satl

import info.nightscout.androidaps.plugins.pump.insight.utils.ByteBuf

class VerifyConfirmRequest : SatlMessage() {

    override val data: ByteBuf
        get() {
            val byteBuf = ByteBuf(2)
            byteBuf.putUInt16LE(PairingStatus.CONFIRMED.id)
            return byteBuf
        }
}